desc/objectives/linear_objectives.py

```diff
@@ -3238,3 +3238,218 @@ def compute(self, params, constants=None):
 
         """
         return params["Psi"]
+
+
+class FixNearAxis(_FixedObjective):
+    """Fixes an equilibrium's near-axis behavior.
```

Comment:
would be good to clarify if this also fixes the axis itself.

IE, do users need to use this in addition to FixAxisR/Z, or are they redundant/conflicting?